src/app/component/project-option/export/export.component.html

```diff
@@ -1,4 +1,12 @@
-<div fxLayout="row wrap" fxFlex="100%" fxLayoutAlign="start start" fxLayoutGap="1em">
+<div style="padding-bottom: 1em">
+  <div fxLayout="row wrap" fxLayoutAlign="space-between start">
+    <div>
+      <mat-slide-toggle [(ngModel)]="configuration.useSymbols" name="use symbols" (change)="callback()">Use Symbols </mat-slide-toggle>
```

Comment:
I don't think looking for change event and calling callback is useful here, these design used to make the renderer to trigger when something changed in the configuration. However since this field only impacts Export feature, I would better not trigger the rendering again.

Reply:
Good point, this is probably down to my misunderstanding of Angular and how these callbacks are made. Thank you for pointing this out. I've updated it now.